src/features/game/expansion/components/Water.tsx

```diff
@@ -215,6 +199,12 @@ export const Water: React.FC<Props> = ({ level }) => {
           />
         </div>
       </MapPlacement>
+
+      <MapPlacement x={offset} y={-offset} width={6}>
```

Comment:
Now that we are interacting with more external contracts I would feel safer if we created a new `CommunityBoundary` component that could catch any potential errors (such as external API/smart contract errors).

In this case here I would suggest creating a new wrapper boundary, similar to our `ErrorBoundar.tsx`

<CommunityBoundary>
  <MapPlacement x={offset} y={-offset} width={6}>
ProjectDignityFrogs left={0} top={0} />
      </MapPlacement>

      <ProjectDignitySeals isGarden={false} />
</CommunityBoundary>
```


```
// CommunityBoundary.tsx
class CommunityBoundary extends Component<Props, State> {
  public state: State = {
    hasError: false,
  };

  public static getDerivedStateFromError(_: Error): State {
    // Update state so the next render will show the fallback UI.
    return { hasError: true };
  }

  public render() {
    if (this.state.hasError) {
      return null;
    }

    return this.props.children;
  }
}
```

Reply:
Hi @adamhannigan great idea to catch these unexpected errors 👌🏻 
I've now added this `CommunityBoundary.tsx` component and imported it to `Water.tsx` 😄 
Cheers!